Sources/HiPMobile/packages.config

```diff
@@ -32,4 +32,5 @@
   <package id="System.Threading" version="4.3.0" targetFramework="portable45-net45+win8" />
   <package id="Unity" version="4.0.1" targetFramework="portable45-net45+win8" />
   <package id="Xam.Plugins.Settings" version="2.5.1.0" targetFramework="portable45-net45+win8" />
+  <package id="Xamarin.Forms" version="2.4.0.280" targetFramework="portable45-net45+win8" />
```

Comment:
As noted above already, is this needed?

Reply:
It doesnt seems to be needed. Removed in next commit

Reply:
I still see it [here](https://github.com/HiP-App/HiP-Forms/blob/933ef384945ca586866a86cf02678f054b5eb485/Sources/HiPMobile/HiPMobile.csproj#L316)